CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ endif()
 include(CTest)
 
 add_subdirectory(src)
+add_subdirectory(test)
```

Comment:
I already include the test directory (see below) but only if the project is at the top level. We *could* just remove that check instead.